test/server/dom-shims.js

```diff
@@ -1,7 +1,8 @@
 /* eslint-env mocha */
-
+/* eslint-disable no-unused-expressions */ // complains about .to.be.ok
```

Comment:
can you set this in an `overrides` section of the eslintrc rather than one-off in this file? make it so anything in the test/ dir (or at least test/server/) automatically uses `mocha` env and turns off `no-unused-expressions`

Reply:
I took some liberties in unifying to a single `eslintrc` in the root directory and to replace the one one `test/browser`, I imported sinon/expect instead of declaring it global. is this cool? https://github.com/mixpanel/panel/pull/110/commits/3dad3a7d6d597e87ce951f8ddc6aa83e8e58ea0e